aries-site/src/examples/components/namevaluelist/data.js

```diff
@@ -91,3 +106,8 @@ export const serverData = {
   Version: '1.18',
   Nodes: '3 Control Plane, 5 workers',
 };
+
```

Comment:
For the `serverData` above, we should modify a couple of things:

1. In a real world scenario, the data won't be containing JSX. It will just be a value. We should modify this to just be 'healthy'. We should then have some sort of `HEALTH_MAP` in the example implementation which maps the 'healthy' to the correct icon.
2. The icon is currently size="medium", should be "small"